src/org/zalando/stups/pierone/api_v1.clj

```diff
@@ -44,13 +49,44 @@
         (ring/header "X-Docker-Token" (get (:tokeninfo request) "access_token" "AnonFakeToken"))
         (ring/header "X-Docker-Endpoints" (get-in request [:headers "host"])))))
 
+
+(defcommand
+  fetch-service-team
+  [kio-url access-token user-id]
+  ; the service user's uid is usually "stups_<application-id>"
+  (let [app-id (last (str/split user-id #"_"))]
+       (:team_id (:body (http/get (r/conpath kio-url "/apps/" app-id)
```

Comment:
The nicer way would be to use our Service User API, if feasible.

~~~ clojure
(let [team-id (:owner (http/get (r/conpath service-user-url "/services/" user-id])
~~~